core/kazoo_services/doc/bookkeepers.md

```diff
@@ -24,7 +24,57 @@ The first [bookkeeper module](https://github.com/2600hz/kazoo/blob/master/core/k
 
 ## Define your own bookkeeper<a id="orgheadline2"></a>
 
-Now, it is possible to define your own bookkeeper service to interact with Kazoo. Kazoo will send your server a request via HTTP and, based on the response, update the requesting account accordingly. Let's take a look.
+Now, it is possible to define your own bookkeeper service to interact with Kazoo. Kazoo will send your server a request via HTTP and, based on the response, update the requesting account accordingly.
+
+Define HTTP bookkeeper by creating the following `system_config > services.http_sync` document in the DB.
+
+```json
+{
+   "_id": "services.http_sync",
+   "default": {
+       "authorization_header": "123abc",
+       "http_url": "http://my-bookeeper-url/"
+   }
+}
+```
+Assign the http bookkeeper to the master account in the `system_config > services` document
+
+```json
+"default": {
+       "sync_services": true,
+       "support_billing_id": true,
+       "should_save_master_audit_logs": false,
+       "master_account_bookkeeper": "kz_bookkeeper_http",
+       "scan_rate": 20000,
+       "sync_buffer_period": 600
+   },
+   ```
+After refreshing the db settings and adding a SIP device, Kazoo will contact the remote bookkeeper with the new information. 
```

Comment:
@powerpbx thanks for the update. after reviewing it again, i think the `After refreshing the db settings` is not accurate. it should be `After assigning the http bookkeeper to the master account, whenever a billable change occurs (device, user, ...), Kazoo will .... ` 

there should be no need to refresh db settings as they should be automatically be used after the creation/update of the configuration/services docs.